Source/WebCore/platform/network/soup/ResourceRequestSoup.cpp

```diff
@@ -140,6 +140,23 @@ void ResourceRequest::updateSoupMessageBody(SoupMessage* soupMessage, BlobRegist
 
 }
 
+GRefPtr<GInputStream> ResourceRequest::createBodyStream() const
+{
+    auto formData = httpBody();
+    if (!formData || formData->isEmpty())
+        return nullptr;
+
+    auto resolvedFormData = formData->resolveBlobReferences();
+    uint64_t length = 0;
+    for (auto& element : resolvedFormData->elements())
+        length += element.lengthInBytes();
+
+    if (!length)
+        return nullptr;
+
+    return webkitFormDataInputStreamNew(WTFMove(resolvedFormData));
```

Comment:
This can still be simplified. We don't need to count up to the full length because it's not used:

```suggestion
    auto resolvedFormData = formData->resolveBlobReferences();
    for (auto& element : resolvedFormData->elements())
        if (element.lengthInBytes() > 0)
            return webkitFormDataInputStreamNew(WTFMove(resolvedFormData));

    return nullptr;
```

Reply:
I'll need to add braces because `check-webkit-style` isn't happy with multi line control clauses without it.